src/class-fields/propname-constructor.case

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Static class fields cannot have PropName 'prototype'
+info: |
+  // This test file tests the following early error:
```

Comment:
this might look weird, but its to make the generated test file's info text a little easier to read. The case file info text is concatenated after the template file text, and it looks confusing without this clarification.

At least, I think it's less confusing this way ;)